### PR TITLE
Fix snippet in `docs/apalache/parameters.md`

### PR DESCRIPTION
**Pull request by BGR360**
The module actually goes until line 15: https://github.com/informalsystems/apalache/blob/0795560e3382fc22b5b5ecabc02346bd5b2849d8/test/tla/y2k_instance.tla#L1-L15

Btw, at a quick glance it doesn't appear that the contributing guidelines mention anything about PRs for docs.

### CODE REVIEW

**Comment by shonfeder** [comment]
Thanks very much for the contribution @BGR360! :pray: 

> Btw, at a quick glance it doesn't appear that the contributing guidelines mention anything about PRs for docs.

Is there any particular information you think it would be useful for us to add there on the topic?

**Comment by BGR360** [comment]
> Is there any particular information you think it would be useful for us to add there on the topic?

@shonfeder that's your call as the maintainer(s)! If there's nothing in particular you want out of these sorts of contributions, then I'd just explicitly say that. Like "for contributions not relating to the source code (such as changes to documentation), none of the other stuff mentioned here matters."

**Review by shonfeder — APPROVED** [review]

**Comment by shonfeder** [comment]
I’ll review the contributing guidelines with an eye to the docs :)

Thanks again!

**Pull request merged**
